### PR TITLE
Short name links

### PR DESCRIPTION
**Pull request by jefflau**

### CODE REVIEW

File: src/components/SingleName/ShortName.js [hunk 1]
```diff
@@ -36,20 +37,20 @@ const InnerWrapper = styled('div')`
...
+  const link = `https://opensea.io/assets/0xFaC7BEA255a6990f749363002136aF6556b31e04/${labelhash}`
   return (
     <ShortNameContainer>
       <InnerWrapper>
         <p>
-          Short names are not currently on auction, but they can be reserved at{' '}
-          <a href="https://reserve.ens.domains">reserve.ens.domains</a>. The
-          auctions will begin soon after the reservation process.
+          Short names are currently on auction, but they can be reserved at{' '}
+          <a href={link}>OpenSea</a>. The auctions will begin soon after the
```
Comment (makoto) [inline src/components/SingleName/ShortName.js]: Isn't it better to open new tab?

Comment (makoto) [reply src/components/SingleName/ShortName.js]: It also shows 404 when clicking the link to https://opensea.io/assets/0xFaC7BEA255a6990f749363002136aF6556b31e04/f06ea683845d8994338d98e7296850612317647cdb1db3d1051f24bafdefe9f7 . Does this not exist yet?

File: src/components/SingleName/ShortName.js [hunk 2]
```diff
@@ -36,20 +37,20 @@ const InnerWrapper = styled('div')`
...
+          reservation process.
         </p>
-        <ExternalButtonLink
-          href="https://reserve.ens.domains"
-          type="hollow-primary"
-        >
-          Reserve Now
+        <ExternalButtonLink href={link} type="hollow-primary">
+          Bid Now
```
Comment (makoto) [inline src/components/SingleName/ShortName.js]: Same as above

File: src/routes/Home.js [hunk 3]
```diff
@@ -302,8 +302,8 @@ export default props => (
     </Hero>
     <ShortNameAuctions>
       <h3>
-        The short name reservations are in progress at&nbsp;
-        <a href="https://reserve.ens.domains">reserve.ens.domains</a>
+        The short name auctions are in progress at&nbsp;
+        <a href="https://opensea.io/ens-landing">opensea.io/ens-landing</a>
```
Comment (makoto) [inline src/routes/Home.js]: Same as above. better to open new tab

File: src/components/SingleName/ShortName.js (hunk 1 above)
Comment (jefflau) [reply src/components/SingleName/ShortName.js]: Doesn't exist yet

File: src/routes/Home.js (hunk 3 above)
Comment (jefflau) [reply src/routes/Home.js]: Don't think it's necessary to open a new tab since the manager does not support the auction

Comment (makoto) [reply src/routes/Home.js]: We open new tab when people click external link so it is consistent to provide the same behavior. It's irrelevant whether the auction starts or not.

Comment (jefflau) [reply src/routes/Home.js]: It is relevant. Because on mobile it will open a new tab and that is not a great experience.

**Pull request merged**